vhdbuilder/packer/install-dependencies.sh

```diff
@@ -326,11 +326,7 @@ MULTI_ARCH_SWIFT_CNI_VERSIONS="
 1.4.32
 "
 
-if [[ $(isARM64) == 1 ]]; then
-  SWIFT_CNI_VERSIONS="${MULTI_ARCH_SWIFT_CNI_VERSIONS}"
-else
-  SWIFT_CNI_VERSIONS="${AMD64_ONLY_SWIFT_CNI_VERSIONS} ${MULTI_ARCH_SWIFT_CNI_VERSIONS}"
-fi
+SWIFT_CNI_VERSIONS="${MULTI_ARCH_SWIFT_CNI_VERSIONS}"
```

Comment:
nit: you can remove this since it's redefining the same existing variable, just update the references below